model/client/SynchronisationClient.php

```diff
@@ -250,6 +250,22 @@ public function sendSyncFinishedConfirmation(array $syncParams)
         return $this->decodeResponseBody($response);
     }
 
+    /**
+     * Send confirmation to the central server about failed synchronization on the client.
+     *
+     * @param array $syncParams
+     * @return array
+     * @throws \common_Exception
+     */
+    public function sendSyncFailedConfirmation(array $syncParams)
+    {
+        $url = '/taoSync/SynchronisationApi/confirmSyncFailed';
+        $method = \Request::HTTP_POST;
```

Comment:
redundant variable, without this variable the code seems cleaner

Reply:
Fixed